### DISCUSSION TITLE
Welcome to AirBoardGame Discussions!

### DISCUSSION THREAD

**Discussion by jrmi** [jrmi/airboardgame#355] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.

  To get started, comment below.

**Comment by Ribakys** [comment]
Hi :)

**Comment by jrmi** [comment]
Hey you can join us on discord https://discord.gg/EsZGJ5h6UA! The server is AirBoardGame.
